### ISSUE TITLE
 Exporting the operator 'aten::fused_moving_avg_obs_fake_quant' to ONNX opset version 14 is not supported.

### ISSUE THREAD

**Issue by seamoonlight-YBY** [pytorch/pytorch#108828]
### 🐛 Describe the bug

# my code:
from torch.quantization import QuantStub, DeQuantStub
import torch.quantization
class Net(nn.Module):
    def __init__(self):
        super(Net, self).__init__()
        self.conv1 = nn.Conv2d(3, 6, 5)
        self.bn1 = nn.BatchNorm2d(6)
        self.relu1 = nn.ReLU()
        self.quant = QuantStub()
        self.dequant = DeQuantStub()
    def forward(self, x):
        x = self.quant(x)
        x = self.relu1(self.bn1(self.conv1(x)))
        x = self.dequant(x)
        return x
    def fuse_model(self):
        torch.quantization.fuse_modules(self, ['conv1', 'bn1', 'relu1'], inplace=True)
net = Net()
print('===== Before fuse_model:')
print(net)
print('===== After fuse_model:')
net.train()
print(net)
print('===== Setting qconfig:')
net.qconfig = torch.quantization.get_default_qat_qconfig('qnnpack')
print('===== After torch.quantization.prepare:')
torch.quantization.prepare_qat(net, inplace=True)
print(net)
input_fp32 = torch.randn((1, 3, 48, 48))
output_x = net(input_fp32)
torch.onnx.export(net, input_fp32, './tmp.onnx', export_params=True, opset_version=14, input_names=['input'],
                 operator_export_type=torch.onnx.OperatorExportTypes.ONNX_FALLTHROUGH,
                 output_names=['output'],
                 keep_initializers_as_inputs=True ,
                 )

# bug

fake_quant_enabled=tensor([1]), observer_enabled=tensor([1]), scale=tensor([1.]), zero_point=tensor([0], dtype=torch.int32), dtype=torch.quint8, quant_min=0, quant_max=255, qscheme=torch.per_tensor_affine, reduce_range=False
      (activation_post_process): MovingAverageMinMaxObserver(min_val=inf, max_val=-inf)
    )
  )
  (relu1): ReLU()
  (quant): QuantStub(
    (activation_post_process): FusedMovingAvgObsFakeQuantize(
      fake_quant_enabled=tensor([1]), observer_enabled=tensor([1]), scale=tensor([1.]), zero_point=tensor([0], dtype=torch.int32), dtype=torch.quint8, quant_min=0, quant_max=255, qscheme=torch.per_tensor_affine, reduce_range=False
      (activation_post_process): MovingAverageMinMaxObserver(min_val=inf, max_val=-inf)
    )
  )
  (dequant): DeQuantStub()
)
torch.Size([1, 6, 44, 44])
============= Diagnostic Run torch.onnx.export version 2.0.0+cu117 =============
verbose: False, log level: Level.ERROR
======================= 0 NONE 0 NOTE 0 WARNING 4 ERROR ========================
ERROR: missing-standard-symbolic-function
Exporting the operator 'aten::fused_moving_avg_obs_fake_quant' to ONNX opset version 14 is not supported. Please feel free to request support or submit a pull request on PyTorch GitHub: https://github.com/pytorch/pytorch/issues.
None
<Set verbose=True to see more details>

ERROR: missing-standard-symbolic-function
Exporting the operator 'aten::fused_moving_avg_obs_fake_quant' to ONNX opset version 14 is not supported. Please feel free to request support or submit a pull request on PyTorch GitHub: https://github.com/pytorch/pytorch/issues.
None
<Set verbose=True to see more details>

ERROR: missing-standard-symbolic-function
Exporting the operator 'aten::fused_moving_avg_obs_fake_quant' to ONNX opset version 14 is not supported. Please feel free to request support or submit a pull request on PyTorch GitHub: https://github.com/pytorch/pytorch/issues.
None
<Set verbose=True to see more details>

### Versions

torch.__version__ ='2.0.0+cu117'
